generators/add-events/index.js

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const path = require('path')
+const Generator = require('yeoman-generator')
+
+const { atLeastOne } = require('../../lib/utils')
+
+const { eventCodes } = require('../../lib/constants')
+
+// we have one actions generator per service, an action generator could generate different types of actions
+const eventCodeToActionGenerator = {
```

Comment:
I don't think you need eventCodes as we do for the sdkCodes. Sdk codes are actual codes used by console for describing adobe APIs.
Here you can remove the constant and the two maps